pandas/io/sas/sas7bdat.py

```diff
@@ -54,12 +54,16 @@ def _convert_datetimes(sas_datetimes: pd.Series, unit: str) -> pd.Series:
         if unit == "s":
             s_series = sas_datetimes.apply(
                 lambda sas_float: datetime(1960, 1, 1) + timedelta(seconds=sas_float)
+                if pd.notnull(sas_float)
```

Comment:
can you use `nona` and import it at the top.

Reply:
done